src/notes/service/NoteSender.swift

```diff
@@ -33,13 +33,21 @@ class DefaultNoteSender: NoteSender {
     }
     
     func sendNote(note: Note, viewController: UIViewController) {
-        // TODO - Figure out how to send the note as an email where
-        // the title is the subject and the note body is the email body
         
-        let alert = UIAlertController(title: "Coming soon", message: "Feature not yet implemented.", preferredStyle: .alert)
+        // set up activity view controller
+        let noteToShare = [note.body]
+        
+        let activityViewController = UIActivityViewController(activityItems: noteToShare as [Any], applicationActivities: nil)
```

Comment:
Is this cast to `as [Any]` array necessary? If not, you should be able to drop it. `[Any]` is similar to `Object[]` in Java. If it is required, it might a little but more clear if you cast the variable directly like `let noteToShare = [note.body] as [Any]`